public/app/features/admin/state/actions.ts

```diff
@@ -142,14 +143,21 @@ export function loadUserSessions(userId: number): ThunkResult<void> {
       return;
     }
 
-    const tokens = await getBackendSrv().get(`/api/admin/users/${userId}/auth-tokens`);
+    const resp = await lastValueFrom(
+      getBackendSrv().fetch<UserSession[]>({
+        url: `/api/admin/users/${userId}/auth-tokens`,
+      })
+    );
+
+    const tokens = resp.data;
     tokens.reverse();
-    const sessions = tokens.map((session: UserSession) => {
+
+    const sessions = tokens.map((session) => {
       return {
         id: session.id,
         isActive: session.isActive,
         seenAt: dateTimeFormatTimeAgo(session.seenAt),
-        createdAt: dateTimeFormat(session.createdAt, { format: 'MMMM DD, YYYY' }),
+        createdAt: new Date(session.createdAt),
```

Comment:
TODO: I made this change to the _admin_ model, but I havent updated the component used on the admin page.